tools/ngsderive.wdl

```diff
@@ -35,3 +37,33 @@ task infer_strand {
         File strandedness_file = stdout()
     }
 }
+
+task read_length {
+    input {
+        File bam
+        File bai
+        Int max_retries = 1
+        Int memory_gb = 5
+    }
+
+    Float bam_size = size(bam, "GiB")
+    Int disk_size = ceil(((bam_size) * 2) + 10)
+ 
+    command {
+        mv ~{bai} ~{bam}.bai 
+        ngsderive readlen ~{bam} | awk 'NR > 1' | cut -d$'\t' -f5
```

Comment:
One thing that Andrew F. updated on the QC branch was parsing the output from ngsderive in a way that doesn't fail if other things are written to stdout. For example, the cluster may print job information on stdout and this gets read by the workflow and will cause a failure if it tries to use that output.

One way is to write the output of ngsderive to a file and then use that file instead of stdout(). 
```suggestion
        ngsderive readlen ~{bam} | awk 'NR > 1' | cut -d$'\t' -f5 > outfile.txt
```